at.medevit.ch.artikelstamm.elexis.common.tests/src/at/medevit/ch/artikelstamm/elexis/common/ArtikelstammImporterTest.java

```diff
@@ -156,7 +111,14 @@ public void testImportOverridingIsOkay() throws IOException{
 		overridden = ArtikelstammItem.findByEANorGTIN(gtinWithPkgSizeOverride);
 		assertEquals(NEW_PKG_SIZE, overridden.getVerpackungsEinheit());
 
-		// TODO: checkResettingVerpackungsEinheit(gtinWithPkgSizeOverride, old_pkg_size);
+		checkResettingVerpackungsEinheit(gtinWithPkgSizeOverride, OLD_PKG_SIZE);
+
+		// Check an article no long present
+		ArtikelstammItem onlyInFirst = ArtikelstammItem.loadByPHARNo(pharOnlyInFirst);
+		if ( onlyInFirst != null  )
+			log.debug("onlyInFirst {} {} isBlackBoxed {} isDeleted {}  ", onlyInFirst.getDSCR(), onlyInFirst.getPHAR(), onlyInFirst.isBlackBoxed(), onlyInFirst.isDeleted());
+		// Next Check fails why?
+		// TODO: assertTrue(onlyInFirst == null || onlyInFirst.isDeleted() );
```

Comment:
Hier ist mir unklar, ob das ein Fehler in der Test-Umgebung oder beim Import ist, dass ich diesen Artikel hier nochmals finde. Da wäre es gut, wenn Marco mal Zeit hätte, den UnitTest darauf zu überprüfen.